### DISCUSSION TITLE
why error.message its error when i try to handling max value.

### DISCUSSION THREAD

**Discussion by ilhamgepe** [react-hook-form/react-hook-form#8519] [Q&A] [answered]
i watch Bill Luo youtube here [React Hook Form V7 - Get Started](https://youtu.be/DN8v7_RbVlc)
but when  i try to handling an error for maximal age.  i got error in my vscode. but everything is working well. 

this my code
```
      <label htmlFor="age" className="mb-5">
        <p>Age: </p>
        <input
          className="w-full p-2 mt-3 rounded-md outline-none"
          {...register("age", {
            valueAsNumber: true,
            required: "are you ancient?",
            max: { value: 120, message: "Are you imortal?" }
          })}
          type="number"
          id="age"
        />
      </label>
      {/* here the error. getting error. but everything working well */}
      {errors?.age && <p>{errors?.age?.message}</p>} // message in {errors?.age?.message}
```

this is my [codesandbox](https://codesandbox.io/s/strange-ardinghelli-yinnk6?file=/components/TestHookForm.tsx:1399-1841)
same error at components > TestHookForm.tsx = in Line 65

sorry i don't know english grammar. i hope u understand.

**Comment by bluebill1049** [comment, marked as answer]
fixed: https://codesandbox.io/s/little-voice-ntq62i

```diff
type FormValue = {
  firstName: string;
  lastName: string;
- age: Number;
+ age: number;
  gender: String;
  developer: Boolean;
};
```

**Reply by ilhamgepe** [reply]
sorry to waste your time. I'm just learning typescript. I really appreciate it <3

**Reply by bluebill1049** [reply]
no worries at all.
